packages/shared-ui/src/flowHandler/processHandler.ts

```diff
@@ -91,6 +91,21 @@ export class ProcessHandler {
       return res;
     }
 
+    const block = res.val.blockBody.block;
```

Comment:
Why didn't we add this logic inside the init function of ProcessService? It already has some logic that can trigger the creation of a new process even if an existing one would be available "isProcessInitial") => It would be more natural to me if we added this override in a similar place. Additionally, this has the benefit that we don't call "authProcessService.init" twice (performance-wise I won't make a difference but it might confuse the reader)